### ISSUE TITLE
Can not cd into ~

### ISSUE THREAD

**Issue by GenericError** [GenericError/Project-Shell#11]
You can not `cd` into your home directory with `cd ~`

**Comment by GenericError** [comment]
Alrighty, I'll get to work on that

**Comment by GenericError** [comment]
Here is some example output showing the issue:

```
user@ShellBookAir Project-Shell$ cd ~
~ is not a directory.
```

**Comment by GenericError** [comment]
Here is the output in the testing code so far:

```
user@ShellBookAir Project-Shell$ cd ~
user@ShellBookAir user$
```

**Comment by GenericError** [comment]
The issue was fixed in the commit `1cd2aff` in the branch `complete-restructure`. Once the full restructure is complete, we can merge with master.

As it stands, the issue is fixed in this branch so it will be closed.
